packages/terra-application-utility/src/terra-dev-site/test/application-utility/UtilityMenuCommon.scss

```diff
@@ -0,0 +1,15 @@
+@import './ApplicationUtilityTestCommon';
+
+:local {
+  .avatar-wrapper {
+    @extend %common-avatar-wrapper;
+  }
+
+  .avatar {
+    @extend %common-avatar;
+  }
+
+  .content-wrapper {
+    @extend %common-content-wrapper;
```

Comment:
Lets directly add the styles that we are using here instead of using the @extends syntax. I think this add extra complexity to these examples that aren't needed.

Reply:
Done